### ISSUE TITLE
ffa waste

### ISSUE THREAD

**Issue by V-ktor** [redeclipse/world#34]
Here is a new map I recently made. It was supposed to be a small or medium sized ffa map but turned out to be rather large. Some of the new assets I made were used.
The top layer is large open area while the lower layers are chaotic and cramped and the layout is confusing. I really don't know why I thought people could like it.
The map also features an improvised lift using pushers. I should probably just remove it and add something like smoke particles instead.
There are affinity entities but it won't work well for game modes other than dm. Bots stick to the room with the stairs. It was even worse before I removed some stairs and added conveyors. Now they at least consider going to adjacent rooms.

[waste_disposal_site.zip](https://github.com/red-eclipse/world/files/1674605/waste_disposal_site.zip)

![20180126133006](https://user-images.githubusercontent.com/12966454/35524811-08f6c2c2-0523-11e8-8fad-fa3433937fdd.jpg)
![20180126132954](https://user-images.githubusercontent.com/12966454/35524819-0c68f362-0523-11e8-854e-bf98ed1229f8.jpg)
![20180126132912](https://user-images.githubusercontent.com/12966454/35524837-1a678dc0-0523-11e8-93c5-79f817889012.jpg)

**Comment by V-ktor** [comment]
I updated this map. It is less cramped and more open now. There are no more problems with the bots either.
I also added the necessary entities for gauntlet mode. Some paths were blocked off by map models, not the best way to do this but it is now the only map that supports gauntlet mode. The finish checkpoint is in the room that is only reachable by teleports. I also moved the rocket to this room.
[waste_disposal_site.zip](https://github.com/red-eclipse/world/files/2077530/waste_disposal_site.zip)
![20180606194718](https://user-images.githubusercontent.com/12966454/41056203-aece2848-69c3-11e8-9e16-bb83d5d105f7.jpg)
![20180606194750](https://user-images.githubusercontent.com/12966454/41056206-b3b8bc9c-69c3-11e8-8f1f-b4a9f8fdbba4.jpg)
